library/src/morpheus-master/sass/themes/_light.scss

```diff
@@ -326,13 +363,13 @@
     --tool-menu-item-hidden-hover-icon-color:                   var(--tool-menu-item-hover-icon-color);
     --tool-menu-item-hidden-hover-background-color:             var(--tool-menu-item-hover-background-color);
 
-    --tool-menu-item-selected-text-color: var(--sakai-color-blue-8);
-    --tool-menu-item-selected-icon-color: var(--sakai-color-blue-6);
-    --tool-menu-item-selected-background-color: var(--sakai-color-blue-0);
+    --tool-menu-item-selected-text-color: var(--sakai-text-color-3);
+    --tool-menu-item-selected-icon-color: var(--sakai-text-color-3);
+    --tool-menu-item-selected-background-color: var(--sakai-active-color-1);
 
     --tool-menu-item-selected-hover-text-color: var(--tool-menu-item-selected-text-color);
     --tool-menu-item-selected-hover-icon-color: var(--tool-menu-item-selected-icon-color);
-    --tool-menu-item-selected-hover-background-color: var(--sakai-color-blue-1);
+    --tool-menu-item-selected-hover-background-color: var(--sakai-active-color-2);
```

Comment:
Begin making use of the new sakai-active colors for easier theming. If this PR is merged, tool menu and other areas will receive similar treatment as favorites did above.